Newspack/Newspack/Controllers/MainNavController.swift

```diff
@@ -31,23 +61,46 @@ final class MainNavController: UINavigationController {
             return
         }
 
-        let menuController = MainStoryboard.instantiateViewController(withIdentifier: .menu)
-        let folderController = MainStoryboard.instantiateViewController(withIdentifier: .folderList)
-        let navController = UINavigationController(rootViewController: folderController)
-
-        let controller = SidebarContainerViewController(mainViewController: navController, sidebarViewController: menuController)
-        controller.delegate = self
-        controller.view.backgroundColor = .basicBackground
-        controller.modalTransitionStyle = .crossDissolve
-        setViewControllers([controller], animated: true)
+        setViewControllers([sidebarContainerController], animated: true)
 
         presentedViewController?.dismiss(animated: true, completion: nil)
     }
+
+}
+
+// MARK: - State Restoration
+
+extension MainNavController: UIViewControllerRestoration {
+    static let sidebarControllerIdentifier = String(describing: type(of: SidebarContainerViewController.self))
+    static let storyNavIdentifier = String(describing: type(of: StoryNavigationController.self))
+    static let menuControllerIdentifier = String(describing: type(of: MenuViewController.self))
```

Comment:
[Maaaaybe this is useful](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/NSObject+Helpers.swift#L10) (OR not!!!)

Reply:
Yeah. Let's do that. Good suggestion

Reply:
Okay... what's funny... is I already had added that extension... and just forgot! 🤦